package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.3.5",
   "description": "AST utility module for statically analyzing JSX",
   "main": "lib/index.js",
+  "files": [
+    "lib"
```

Comment:
🤷‍♂️ I see where you're coming from but feel differently. Let's agree to disagree. 😀